src/index.html

```diff
@@ -8,6 +8,22 @@
         .services-table .ant-table td {
           white-space: nowrap;
         }
+        .outsideWrapper{ 
```

Comment:
can we move all of the service specific css into css files for those services that are imported locally within the service file itself?